src/instructlab/configuration.py

```diff
@@ -1356,7 +1356,7 @@ def storage_dirs_exist() -> bool:
 
 # {"GPU Name": ({ "gpu_count": NUM_GPUS, "vram_and_config": {"vram": TOTAL_VRAM, "config": TRAIN_CONFIG}})...}
 PROFILE_MAPPINGS = {
-    "L40s": (
+    "L40S": (
```

Comment:
Would it be better to put all the keys in as uppercase and always upper(gpu_name) before checking in the dict?  It looks like this is the only one that wasn't already uppercase and so the only other change would be to add the upper() call to the get() (add a comment).